akka-docs/rst/scala/code/docs/persistence/PersistenceDocSpec.scala

```diff
@@ -179,11 +185,6 @@ object PersistenceDocSpec {
       override def receiveCommand: Receive = ???
     }
 
-    //#snapshot-criteria
-    persistentActor ! Recovery(fromSnapshot = SnapshotSelectionCriteria(
-      maxSequenceNr = 457L,
-      maxTimestamp = System.currentTimeMillis))
-    //#snapshot-criteria
```

Comment:
this was wrong, since a previous PR we can not just tell random actors to recover, it is only via overriding `recovery`